app/jobs/import_url_job.rb

```diff
@@ -92,7 +85,7 @@ def send_error(error_message)
     # @param f [IO] the stream to write to
     def write_file(uri, f, headers)
       retriever = BrowseEverything::Retriever.new
-      uri_spec = { 'url' => uri }.merge(headers)
+      uri_spec = ActiveSupport::HashWithIndifferentAccess.new(url: uri, headers: headers)
```

Comment:
Does the browse everything `Retriever` need this to have indifferent access? or can it do alright with `String` keys?

Reply:
It looks like BE only expects string keys as far as I can tell.

Reply:
`Typhoeus::Request` seemed to prefer symbols for keys (please see https://github.com/samvera/browse-everything/blob/master/lib/browse_everything/retriever.rb#L39, https://github.com/samvera/browse-everything/blob/master/lib/browse_everything/retriever.rb#L138 and https://github.com/typhoeus/typhoeus#introduction), and there were some legacy areas of the code base where strings were still used (https://github.com/samvera/browse-everything/blob/master/lib/browse_everything/retriever.rb#L95).  I can address this with a patch release for `browse-everything` (https://github.com/samvera/browse-everything/issues/290) and issue a separate PR resolving this.